backend/src/apiserver/server/run_log_server.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 Google LLC
```

Comment:
nit: `2020`
Also, we are open to marking these files as `Copyright 2020 The Kubeflow Authors` if you'd prefer that.